advanced-achievements-plugin/src/main/resources/config.yml

```diff
@@ -277,16 +277,15 @@ Places:
           Display: Become the Stone God!
 
 # When a specific block is broken (specify material name, lower case).
-# Below, the notation sand:1 means sand item with metadata 1 (red sand).
 Breaks: 
-  sand:1:
+  red_sand:
```

Comment:
Should we remove this example entirely? It's kind of similar to "stone" in the same category.

Reply:
I agree....sand is not needed if we find ice/snow or something else to demonstrate combos